engines/tinsel/music.cpp

```diff
@@ -726,6 +726,10 @@ void PCMMusicPlayer::stopPlay() {
 int PCMMusicPlayer::readBuffer(int16 *buffer, const int numSamples) {
 	Common::StackLock slock(_mutex);
 
+	// Workaround for v3 to prevent deadlock due to missing chunk
+	if (!_curChunk && (TinselVersion == 3) && _state == S_MID)
```

Comment:
Might be worth dumping a debug message at least once, so this gets a proper fix eventually.